src/components/PlantsV2/components/PlantingProgressPerZoneCard.tsx

```diff
@@ -26,9 +30,9 @@ export default function PlantingProgressPerZoneCard({ plantingSiteId }: Planting
       plantingSite.plantingZones?.forEach((zone) => {
         let completedPlantingArea = 0;
         zone.plantingSubzones.forEach((sz) => {
-          completedPlantingArea += sz.plantingCompleted ? +sz.areaHa : 0;
+          completedPlantingArea += sz.plantingCompleted ? parse(sz.areaHa) : 0;
```

Comment:
you can avoid the parsing by fetching `(raw)` values in the search.. might be simpler
see example https://github.com/terraware/terraware-web/blob/9a7bd29b56a6ae2d1ab7f4dbd04e4546e20fa4fe/src/services/NurseryBatchService.ts#L23